runtimes/eoapi/stac/eoapi/stac/app.py

```diff
@@ -179,38 +138,42 @@ async def lifespan(app: FastAPI):
 
 # Middlewares
 middlewares = [Middleware(CompressionMiddleware)]
-if api_settings.cors_origins:
+if settings.cors_origins:
     middlewares.append(
         Middleware(
             CORSMiddleware,
-            allow_origins=api_settings.cors_origins,
+            allow_origins=settings.cors_origins,
             allow_credentials=True,
-            allow_methods=api_settings.cors_methods,
+            allow_methods=settings.cors_methods,
             allow_headers=["*"],
         )
     )
 
 api = StacApi(
     app=FastAPI(
-        title=api_settings.name,
+        title=settings.stac_fastapi_title,
+        description=settings.stac_fastapi_description,
         lifespan=lifespan,
-        openapi_url="/api",
-        docs_url="/api.html",
+        openapi_url=settings.openapi_url,
+        docs_url=settings.docs_url,
         redoc_url=None,
         swagger_ui_init_oauth={
             "clientId": auth_settings.client_id,
             "usePkceWithAuthorizationCodeGrant": auth_settings.use_pkce,
         },
     ),
-    title=api_settings.name,
-    description=api_settings.name,
     settings=settings,
     extensions=application_extensions,
-    client=PgSTACClient(pgstac_search_model=post_request_model),
-    items_get_request_model=items_get_request_model,
-    search_get_request_model=get_request_model,
-    search_post_request_model=post_request_model,
-    collections_get_request_model=collections_get_request_model,
+    client=PgSTACClient(  # type: ignore
+        landing_page_id=settings.stac_fastapi_landing_id,
+        title=settings.stac_fastapi_title,
+        description=settings.stac_fastapi_description,
```

Comment:
so we get 
```json
{
  "type": "Catalog",
  "id": "eoapi-devseed-stac",
  "title": "eoAPI-stac",
  "description": "Custom stac-fastapi application for eoAPI Devseed",
  "stac_version": "1.0.0",
```

Reply:
wow these settings come all the way up from `stac.fastapi.config` :astonished: